### PR TITLE
rosanto - quality updates - adding tags, fixing issues

### PR DESCRIPTION
**Pull request by rodrigosantosms**
# Overview/Summary

Quality updates - adding tags, fixing issues with formatting.

## Related Issues/Work Items

N/A

## This PR fixes/adds/changes/removes

rosanto - quality updates - adding tags, fixing issues with formatting.

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Comment by JefferyMitchell** [comment]
@rodrigosantosms Advisor team has asked that all ARG quires contain subscription ID and Resource ID for advisor. Can they update our queries on APRL to do so or would that break our automation?

**Comment by rodrigosantosms** [comment]
> @rodrigosantosms Advisor team has asked that all ARG quires contain subscription ID and Resource ID for advisor. Can they update our queries on APRL to do so or would that break our automation?

All ARGs today already output the ResourceID which contains SubscriptionID too. Yes anybody from Microsoft can update our Queries and submit PRs.

**Review by JefferyMitchell — APPROVED** [review]
Approving with additional reviews from others in the workload team to look at DDOS and Resolver.

File: docs/content/services/compute/virtual-machine-scale-sets/code/vmss-11/vmss-11.kql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Azure Resource Graph Query
+// Find all VMSSs Uniform not using SSD storage for OS disk
+resources
+| where type == "microsoft.compute/virtualmachinescalesets"
+| where properties.orchestrationModeS != "Flexible"
+| where properties.virtualMachineProfile.storageProfile.osDisk.managedDisk.storageAccountType == 'Standard_LRS'
+| project recommendationId = "vmss-11", name, id, tags
```
[the remark this answers is not in the document]
Comment (oZakari) [reply docs/content/services/compute/virtual-machine-scale-sets/code/vmss-11/vmss-11.kql:7]: I know you tested the queries and I don't mean to be a pain. However, as this is a new query and to consistent in our review process could you please add a screenshot?

File: docs/content/services/web/web-app/code/app-6/app-6.kql
Comment (oZakari) [inline docs/content/services/web/web-app/code/app-6/app-6.kql:1]: Can you please add screenshot for this as well?

**Review by oZakari — CHANGES_REQUESTED** [review]

File: docs/content/services/compute/virtual-machine-scale-sets/code/vmss-11/vmss-11.kql (hunk 1 above)
[the remark this answers is not in the document]
Comment (rodrigosantosms) [reply docs/content/services/compute/virtual-machine-scale-sets/code/vmss-11/vmss-11.kql]: Replied internally.

**Review by ehaslett — APPROVED** [review]
LGTM

File: docs/content/services/web/web-app/code/app-6/app-6.kql
Comment (oZakari) [reply docs/content/services/web/web-app/code/app-6/app-6.kql:1]: Rodrigo shared screenshot in Teams.

**Review by oZakari — APPROVED** [review]

**Pull request merged**